ng-dev/pr/common/targeting/labels.ts

```diff
@@ -80,7 +81,18 @@ export async function getTargetLabelsForActiveReleaseTrains(
       // TODO: Consider handling this automatically by checking if the NPM version matches
       // the last-minor. If not, then an exceptional minor might be in progress. See:
       // https://docs.google.com/document/d/197kVillDwx-RZtSVOBtPb4BBIAw0E9RT3q3v6DZkykU/edit#heading=h.h7o5pjq6yqd0
-      branches: () => [nextBranchName],
+      branches: () => {
+        // TODO(devversion): Remove this when we actually support exceptional minors.
+        if (
+          (config as Partial<{pullRequest: PullRequestConfig}>).pullRequest
```

Comment:
I wanted to avoid doing this since the config is just needed for the temporary hack, but I applied the suggestion since it makes sense what you described. Might be able to remove this when we remove this trick.